File: 0020.R
### 0020
options("scipen" = 10000, "digits" = 22)

fac <- function(n){
  if (n != 1){n*fac(n-1)}
  else {n}
}

n <- fac(100)

sum_ <- 0
for (c in 1:nchar(n)){
  sum_ <- sum_ + as.numeric(substr(n, c, c))
}

File: Python/0020.py
### 0020
def fac(n):
    if n != 1:
        return n*fac(n-1)
    else: 
        return n
        
n = str(fac(100))


sum_ = 0
for c in range(0, len(n)):
    print(c)
    sum_ += int(n[c:c+1])
    
sum_
